Cargo.toml

```diff
@@ -1,28 +1,33 @@
 [package]
-name = "async-broadcast"
-version = "0.3.3"
-license = "MIT OR Apache-2.0"
-repository = "https://github.com/smol-rs/async-broadcast"
-documentation = "https://docs.rs/async-broadcast"
-description = "Async broadcast channels"
-readme = "README.md"
-edition = "2018"
-keywords = []
-categories = []
 authors = [
     "Stjepan Glavina <stjepang@gmail.com>",
     "Yoshua Wuyts <yoshuawuyts@gmail.com>",
     "Zeeshan Ali <zeeshanak@gnome.org>",
 ]
+categories = []
+description = "Async broadcast channels"
+documentation = "https://docs.rs/async-broadcast"
+edition = "2018"
+keywords = []
+license = "MIT OR Apache-2.0"
+name = "async-broadcast"
+readme = "README.md"
+repository = "https://github.com/smol-rs/async-broadcast"
+version = "0.3.3"
```

Comment:
These changes are separate logical changes and hence should be in a separate commit w/ its own summary (and possibly rationale).

Reply:
This part is still not relevant to the commit it's in.